changelog

```diff
@@ -1,3 +1,240 @@
+      * Pirate raid deterence no longer looks at whether you have ammo in your launchers. (@TomGoodIdea)
+      * Fleets can now spawn from any section of a ringworld instead of only ever spawning from the same section. (@quyykk)
+      * Cooling is no longer disabled when a ship becomes overheated. (@QuantumShark)
+      * The player now always launches from the same section of the ringworld that they landed on instead of only ever departing from the same section. (@quyykk)
+    * AI:
+      * Ships with only secondary weapons that run out of ammo will now attempt to flee instead of flying straight at their target without any weapons. (@TomGoodIdea)
+      * Escorts that are capable of cloaking can now cloak on their own without input from the player if they are low on health. (@TomGoodIdea)
+      * Ships with the "surveillance" personality will now patrol the system they are in when there are no other ships to scan instead of freezing up and doing nothing. (@tibetiroka)
+  * User interface:
+    * Weapons that use both outfits and fuel as ammunition now properly display the number of shots remaining instead of ignoring the fuel use. (@tibetiroka)
+    * The flagship now has a separate color scheme from friendlies in the HUD overlay. (@warp-core)
+    * Moved the definitions of the ship info hardpoint colors to the interfaces file so that they can be modified without recompiling. (@TomGoodIdea)
+    * Disabling motion blur in the settings now also disables motion blur for the starfield. (@Koranir)
+    * Added support for drawing pointers in the interface game data. (@warp-core)
+    * Increased the saturation of the shields HUD color. (@Quantumshark)
+    * Added a new setting that determines which ships in your fleet will pick up flotsams, if at all. (@OcelotWalrus)
+    * The outfits installed on player ships are now displayed in a list format under all the other ship stats in the shipyard and outfitter, similar to how they're displayed for ships that are sold in the shipyard. (@flowers9)
+    * Changed the "has no shipyard/outfitter" color on the shipyard and outfitter map keys to a darker shade of blue to better differentiate it from the "has shipyard/outfitter" color. (@flowers9)
+    * Added a "landing zoom" UI setting that zooms the camera in or out when landing on or launching from a planet. (@flowers9)
+    * Human engines in the outfitter are now ordered according to the engine set that they come from, from smallest to largest, and listed with steering before thrusters, instead of purely alphabetically. (@warp-core)
+    * The color of a wormhole's link on the map is now also used when that wormhole is a part of the travel plan. (@RisingLeaf, @warp-core)
+    * Increased the amount of motion blur that occurs to the background while hyperspacing. This can be toggled in the settings under the "extended jump effects" preference. (@Koranir)
```

Comment:
```suggestion
    * Increased the amount of motion blur that occurs in the background while hyperspacing. This can be toggled in the settings under the "extended jump effects" preference. (@Koranir)
```